### PR TITLE
Update CopyKiller detection and notes (WIP)

### PR DESCRIPTION
**Pull request by TheRogueArchivist**
Still very WIP, most changes in fact don't appear to work yet.

### CODE REVIEW

File: BinaryObjectScanner/Protection/CopyKiller.cs [hunk 1]
```diff
@@ -1,32 +1,65 @@
 ﻿#if NET40_OR_GREATER || NETCOREAPP
 using System.Collections.Concurrent;
 #endif
+using System;
```
Comment (mnadareski) [inline BinaryObjectScanner/Protection/CopyKiller.cs:4]: Nit: This should go on the first line

File: BinaryObjectScanner/Protection/CopyKiller.cs [hunk 2]
```diff
@@ -1,32 +1,65 @@
...
 using System.Collections.Generic;
 using BinaryObjectScanner.Interfaces;
 using SabreTools.Matching;
+using SabreTools.Serialization.Wrappers;
 
-namespace BinaryObjectScanner.Protection
+namespace BinaryObjectScanner.ProtectionType
```
Comment (mnadareski) [inline BinaryObjectScanner/Protection/CopyKiller.cs:10]: I think you changed this to `ProtectionType` accidentally

File: BinaryObjectScanner/Protection/CopyKiller.cs [hunk 3]
```diff
@@ -1,32 +1,65 @@
...
+            // TODO: Figure out why this check doesn't work.
+            // Found in ""autorun.exe" in CopyKiller V3.64, V3.99, and V3.99a.
+            var name = pex.ProductName;
+            if (name?.StartsWith("CopyKiller", StringComparison.OrdinalIgnoreCase) == true)
+                return $"CopyKiller V3.64+";
 
-                return MatchUtil.GetFirstMatch(file, fileContent, contentMatchSets, includeDebug);
-            }
+            else if (name?.StartsWith("Solidshield Activation Library", StringComparison.OrdinalIgnoreCase) == true)
+                return $"SolidShield Core.dll {pex.GetInternalVersion()}";
```
Comment (mnadareski) [inline BinaryObjectScanner/Protection/CopyKiller.cs:62]: I assume this was a leftover from debugging.

Comment (TheRogueArchivist) [reply BinaryObjectScanner/Protection/CopyKiller.cs:62]: oops

**Review by mnadareski — APPROVED** [review]
CK is OK

**Pull request merged**